dev/build/windows/makecoq_mingw.sh

```diff
@@ -1765,7 +1765,6 @@ function make_addon_menhirlib {
 function make_addon_compcert {
   installer_addon_dependency_beg compcert
   make_menhir
```

Comment:
Maybe I miss something, but the outdated URL seems to be used in `make_menhir` not `make_menhirlib`...

Reply:
What makes you think that? In any case, CI did succeed.

Reply:
Sorry, I found what I was missing.